### PR TITLE
[stable/artifactory] Small fix for ingress 

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory/values.yaml [hunk 1]
```diff
@@ -87,6 +87,8 @@ nginx:
     ## Example: helm install ... --set nginx.service.loadBalancerSourceRanges='{10.10.10.5/32,10.11.10.5/32}'
     loadBalancerSourceRanges: []
     annotations: {}
+    ## Provide static ip address
+    loadBalancerIP:
```
Comment (eldada) [inline stable/artifactory/values.yaml]: Add `loadBalancerIP` to README.md

File: stable/artifactory-ha/values.yaml [hunk 2]
```diff
@@ -183,6 +183,8 @@ nginx:
     ## or pass from helm command line
     ## Example: helm install ... --set nginx.service.loadBalancerSourceRanges='{10.10.10.5/32,10.11.10.5/32}'
     loadBalancerSourceRanges: []
+    ## Provide static ip address
+    loadBalancerIP:
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml]: Add `loadBalancerIP` to README.md

**Review by eldada — CHANGES_REQUESTED** [review]

File: stable/artifactory-ha/values.yaml (hunk 2 above)
Comment (jainishshah17) [reply stable/artifactory-ha/values.yaml]: done

**Review by eldada — APPROVED** [review]

**Pull request closed without merging**